src/main/java/edu/cornell/kfs/module/cg/businessobject/AwardExtendedAttribute.java

```diff
@@ -98,4 +101,28 @@ public void setEverify(boolean everify) {
 	    this.everify = everify;
 	}
 
+    public String getPrimeAgreementNumber() {
+        return primeAgreementNumber;
+    }
+
+    public void setPrimeAgreementNumber(String primeAgreementNumber) {
+        this.primeAgreementNumber = primeAgreementNumber;
+    }
+
+    public String getAutoApproveReason() {
+        return autoApproveReason;
+    }
+
+    public void setAutoApproveReason(String autoApproveReason) {
+        this.autoApproveReason = autoApproveReason;
+    }
+
+    public String getPurchasOrderNumber() {
```

Comment:
please check there is a misspelling in purchase

Reply:
I will fix